packages/builder-util-runtime/src/httpExecutor.ts

```diff
@@ -60,22 +60,40 @@ export class HttpError extends Error {
     this.name = "HttpError"
     ;(this as NodeJS.ErrnoException).code = `HTTP_ERROR_${statusCode}`
   }
+
+  // Check if 500 error
+  isServerError() {
+    return this.statusCode % 500 <= 99
```

Comment:
Nit: this is hard to parse. I think `this.statusCode >= 500 && this.statusCode <= 599` would be easier to read. 🙂

Reply:
Valid. Updating that